types/yup/index.d.ts

```diff
@@ -455,6 +455,7 @@ interface NumberLocale {
     max?: TestOptionsMessage<{ max: number }>;
     lessThan?: TestOptionsMessage<{ less: number }>;
     moreThan?: TestOptionsMessage<{ more: number }>;
+    notEqual?: TestOptionsMessage<{ more: number }>;
```

Comment:
Should be `TestOptionsMessage<{ notEqual: number }>`, but also looks like this may just be left over. There is no `notEqual` method on `NumberSchema` anymore. It looks like it was removed in [this commit](https://github.com/jquense/yup/commit/061e590bc305dbf6dafd14a20bc71d92fd42cd4a#diff-a8628b7684fb33ab3e2921b101c004ab). Looks like a description of reasoning is in [this comment](https://github.com/jquense/yup/pull/63#issuecomment-336444456) from the original PR

Reply:
It makes sense. Thanks @kalley 
I should have found it myself.